test/image_test.py

```diff
@@ -750,6 +750,30 @@ def f(self):
         print("bar!", VARIABLE_6)
 
 
+class FooInstance:
+    bar: str = "5"
+
+    @build()
+    def build_func(self):
+        global VARIABLE_5
+
+        print("foo!", VARIABLE_5)
+        print("static var", FooInstance.bar)
+
+
+def test_image_cls_var_rebuild(client, servicer):
```

Comment:
Would be good to have a couple of the 'negative' cases covered.

1. not rebuilding when not changed
2. not rebuilding when static var not referenced